### PR TITLE
[post-0.4.0 KubeRay release] Update KubeRay versions 

### PR DESCRIPTION
**Pull request by DmitriGekhtman**
Signed-off-by: Dmitri Gekhtman <dmitri.m.gekhtman@gmail.com>

## Why are these changes needed?

Updates KubeRay versions in master.
To be merged after images and Helm charts are built.

## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by DmitriGekhtman** [comment]
@Jeffwan @scarlet25151 I did not modify KubeRay versions in files related to the CLI or API server.
That's up to you guys.

**Review by architkulkarni — APPROVED** [review]

**Review by scarlet25151 — APPROVED** [review]
overall LGTM.

**Comment by DmitriGekhtman** [comment]
@sihanwang41 will make the call on when to merge this.

File: helm-chart/kuberay-apiserver/Chart.yaml [hunk 1]
```diff
@@ -15,10 +15,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.0
-
-# This is the version number of the application being deployed. This version number should be
-# incremented each time you make changes to the application. Versions are not expected to
```
Comment (Jeffwan) [inline helm-chart/kuberay-apiserver/Chart.yaml:21]: Do we want to use helm version to track the upgrade in future?

**Review by Jeffwan — APPROVED** [review]

**Comment by Jeffwan** [comment]
I add ci bot back and let's check how the CI works

**Comment by Jeffwan** [comment]
@scarlet25151 Please update the apiserver versions in a follow up PR.

File: helm-chart/kuberay-apiserver/Chart.yaml (hunk 1 above)
Comment (DmitriGekhtman) [reply helm-chart/kuberay-apiserver/Chart.yaml:21]: Could you elaborate?

The `version` field is still there.

**Pull request merged**